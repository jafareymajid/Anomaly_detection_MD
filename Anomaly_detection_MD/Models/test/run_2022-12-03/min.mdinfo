

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -2.8013E+04     3.1807E+00     5.7554E+01     C         258

 BOND    =     7604.5700  ANGLE   =       43.9360  DIHED      =      142.8778
 VDWAALS =     8954.2002  EEL     =   -45787.0122  HBOND      =        0.0000
 1-4 VDW =       42.3515  1-4 EEL =      971.4710  RESTRAINT  =       14.5762
 EAMBER  =   -28027.6056
