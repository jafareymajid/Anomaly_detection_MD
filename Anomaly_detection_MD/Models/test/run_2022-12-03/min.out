
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 12/03/2022 at 10:21:20

|   Executable path: sander
| Working directory: /mnt/e/MSU/Fall2022/CMSE802/project/scripts/test_software/run_2022-12-03
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: min.in                                                                
| MDOUT: min.out                                                               
|INPCRD: protein.rst7                                                          
|  PARM: protein.parm7                                                         
|RESTRT: min.rst7                                                              
|  REFC: protein.rst7                                                          
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: min.nc                                                                
|MDINFO: min.mdinfo                                                            
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

Energy minimization                                                            
&cntrl                                                                         
imin=1, ntmin=0, maxcyc=200,   ! Minimization, method, number of cycles        
ntpr=100,                         ! Print energies every ntpr steps            
ncyc=100,                                                                      
ntr=1,                          ! Use harmonic cartesian restraints            
cut=9.0,                                                                       
ntwx=0,                                                                        
restraint_wt=10.0,              ! Restraint force kcal/mol                     
restraintmask="(:1-13)&(@CA,N,O)",   !put restraint on protein backbone        
&end                                                                           
END                                                                            

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting new box info from bottom of inpcrd
|  INFO: Old style inpcrd file read

| peek_ewald_inpcrd: Box info found
|Largest sphere to fit in unit cell has radius =    15.883
| New format PARM file being parsed.
| Version =    1.000 Date = 12/03/22 Time = 10:21:12
 NATOM  =    4536 NTYPES =      15 NBONH =    3349 MBONA  =    1174
 NTHETH =     351 MTHETA =     139 NPHIH =     593 MPHIA  =     436
 NHPARM =       0 NPARM  =       0 NNB   =    8942 NRES   =    1091
 NBONA  =    1174 NTHETA =     139 NPHIA =     436 NUMBND =      29
 NUMANG =      58 NPTRA  =      71 NATYP =      21 NPHB   =       1
 IFBOX  =       2 NMXRS  =      23 IFCAP =       0 NEXTRA =    1066
 NCOPY  =       0


|     Memory Use     Allocated
|     Real              465442
|     Hollerith          14701
|     Integer           192180
|     Max Pairs        2414966
|     nblistReal         54432
|     nblist Int        158027
|       Total            14920 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

     BOX TYPE: TRUNCATED OCTAHEDRON

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =     100, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   9.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  10.00000

Energy minimization:
     maxcyc  =     200, ncyc    =     100, ntmin   =       0
     dx0     =   0.01000, drms    =   0.00010

Extra-points options:
     frameon =       1, chngmask=       1

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   38.905   Box Y =   38.905   Box Z =   38.905
     Alpha =  109.471   Beta  =  109.471   Gamma =  109.471
     NFFT1 =   40       NFFT2 =   40       NFFT3 =   40
     Cutoff=    9.000   Tol   =0.100E-04
     Ewald Coefficient =  0.30768
     Interpolation order =    4

    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask (:1-13)&(@CA,N,O); matches    39 atoms
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  3349     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =  3349     0
|      EXTRA_PTS, trim_bonds: num bonds BEFORE trim =  1174     0
|      EXTRA_PTS, trim_bonds: num bonds AFTER  trim =   108     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =   351     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =   351     0
|      EXTRA_PTS, trim_theta: num angle BEFORE trim =   139     0
|      EXTRA_PTS, trim_theta: num angle AFTER  trim =   139     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =   593     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =   593     0
|      EXTRA_PTS, trim_phi:  num diheds BEFORE trim =   436     0
|      EXTRA_PTS, trim_phi:  num diheds AFTER  trim =   436     0

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:     1066

     Sum of charges from parm topology file =   6.00000583
     Assuming uniform neutralizing plasma

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8314E-11   at   2.736960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =    1264287
| TOTAL SIZE OF NONBOND LIST =    1264287


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       9.4533E+03     1.1020E+02     3.7514E+02     O        1861

 BOND    =    20436.9449  ANGLE   =        9.0578  DIHED      =      133.4868
 VDWAALS =      998.9688  EEL     =   -13195.9164  HBOND      =        0.0000
 1-4 VDW =       69.3352  1-4 EEL =     1001.3924  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100      -2.6110E+04     5.0285E+00     1.4195E+02     C         258

 BOND    =     6526.5949  ANGLE   =       44.8069  DIHED      =      140.9489
 VDWAALS =     7242.6237  EEL     =   -41093.8508  HBOND      =        0.0000
 1-4 VDW =       44.2401  1-4 EEL =      972.1358  RESTRAINT  =       12.3908
 EAMBER  =   -26122.5005


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -2.8013E+04     3.1807E+00     5.7554E+01     C         258

 BOND    =     7604.5700  ANGLE   =       43.9360  DIHED      =      142.8778
 VDWAALS =     8954.2002  EEL     =   -45787.0122  HBOND      =        0.0000
 1-4 VDW =       42.3515  1-4 EEL =      971.4710  RESTRAINT  =       14.5762
 EAMBER  =   -28027.6056


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -2.8013E+04     3.1807E+00     5.7554E+01     C         258

 BOND    =     7604.5700  ANGLE   =       43.9360  DIHED      =      142.8778
 VDWAALS =     8954.2002  EEL     =   -45787.0122  HBOND      =        0.0000
 1-4 VDW =       42.3515  1-4 EEL =      971.4710  RESTRAINT  =       14.5762
 EAMBER  =   -28027.6056

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.01 ( 0.04% of Total)
|                Build the list             0.41 (95.80% of List )
|                Other                      0.02 ( 4.20% of List )
|             List time                  0.43 ( 4.16% of Nonbo)
|                   Short_ene time             8.01 (99.36% of Direc)
|                   Other                      0.05 ( 0.64% of Direc)
|                Direct Ewald time          8.06 (82.04% of Ewald)
|                Adjust Ewald time          0.08 ( 0.82% of Ewald)
|                Self Ewald time            0.00 ( 0.01% of Ewald)
|                   Fill Bspline coeffs        0.07 ( 4.38% of Recip)
|                   Fill charge grid           0.15 ( 9.53% of Recip)
|                   Scalar sum                 0.52 (32.65% of Recip)
|                   Grad sum                   0.29 (18.19% of Recip)
|                   FFT time                   0.56 (35.18% of Recip)
|                   Other                      0.00 ( 0.06% of Recip)
|                Recip Ewald time           1.59 (16.22% of Ewald)
|                Virial junk                0.01 ( 0.07% of Ewald)
|                Other                      0.08 ( 0.83% of Ewald)
|             Ewald time                 9.83 (95.83% of Nonbo)
|             Other                      0.00 ( 0.01% of Nonbo)
|          Nonbond force             10.26 (99.34% of Force)
|          Bond/Angle/Dihedral        0.06 ( 0.62% of Force)
|          Other                      0.00 ( 0.05% of Force)
|       Force time                10.32 (100.0% of Runmd)
|    Runmd Time                10.32 (83.49% of Total)
|    Other                      2.04 (16.47% of Total)
| Total time                12.37 (100.0% of ALL  )

| Number of list builds   :          6

| Highest rstack allocated:     268814
| Highest istack allocated:       4536
|           Job began  at 10:21:20.068  on 12/03/2022
|           Setup done at 10:21:20.140  on 12/03/2022
|           Run   done at 10:21:32.435  on 12/03/2022
|     wallclock() was called    7660 times
